platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMapOptions.java

```diff
@@ -644,7 +689,9 @@ public MapboxMapOptions crossSourceCollisions(boolean crossSourceCollisions) {
    */
   @NonNull
   public MapboxMapOptions localIdeographFontFamily(String fontFamily) {
-    this.localIdeographFontFamily = FontUtils.extractValidFont(fontFamily);
+    if (localIdeographFontFamilyEnabled) {
```

Comment:
This flag should be read when options are parsed, not when they are set because it introduces a required order in which the methods need to be called.